<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Triggers</string>
  </property>
  <widget class="QStackedWidget" name="uiTriggerScreens">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="uiTrigger1">
    <widget class="HeaderLabel" name="headerlabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="DecimalSpinBox" name="decimalspinbox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="Label" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>volts</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifiers</string>
     </property>
    </widget>
    <widget class="CheckBox" name="ui1mAPullup">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>1mA Pullup</string>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="ComboBox" name="combobox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Record End Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exposure Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shutter Gating</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frame Sync Output</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Action</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualization</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>341</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255, 244, 123, 105)</string>
     </property>
    </widget>
    <widget class="Backdrop" name="backdrop_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="CheckBox" name="uiInvertOutput_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert Output</string>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="CheckBox" name="uiDebounce_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Debounce</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="CheckBox" name="ui20mAPullup_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>20mA Pullup</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
    </widget>
    <widget class="CheckBox" name="uiInvertInput_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert Input</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
    </widget>
    <widget class="QGraphicsView" name="uiTriggerIcon">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <zorder>backdrop_2</zorder>
    <zorder>headerlabel_2</zorder>
    <zorder>decimalspinbox</zorder>
    <zorder>label</zorder>
    <zorder>headerlabel_3</zorder>
    <zorder>ui1mAPullup</zorder>
    <zorder>headerlabel_4</zorder>
    <zorder>headerlabel_5</zorder>
    <zorder>combobox</zorder>
    <zorder>widget</zorder>
    <zorder>uiInvertOutput_3</zorder>
    <zorder>uiDebounce_2</zorder>
    <zorder>ui20mAPullup_2</zorder>
    <zorder>uiInvertInput_3</zorder>
    <zorder>uiTriggerIcon</zorder>
   </widget>
   <widget class="QWidget" name="uiTrigger2">
    <widget class="HeaderLabel" name="headerlabel_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Action</string>
     </property>
    </widget>
    <widget class="CheckBox" name="uiInvertOutput_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert Output</string>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="CheckBox" name="uiInvertInput_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert Input</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualization</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>341</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255, 244, 123, 105)</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifiers</string>
     </property>
    </widget>
    <widget class="Label" name="label_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>volts</string>
     </property>
    </widget>
    <widget class="DecimalSpinBox" name="decimalspinbox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="CheckBox" name="ui20mAPullup">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>20mA Pullup</string>
     </property>
    </widget>
    <widget class="CheckBox" name="uiDebounce">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Debounce</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="Backdrop" name="backdrop_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="ComboBox" name="combobox_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Record End Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exposure Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shutter Gating</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frame Sync Output</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
    <widget class="QGraphicsView" name="uiTriggerIcon_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <zorder>backdrop_3</zorder>
    <zorder>headerlabel_6</zorder>
    <zorder>uiInvertOutput_2</zorder>
    <zorder>uiInvertInput_2</zorder>
    <zorder>headerlabel_7</zorder>
    <zorder>headerlabel_8</zorder>
    <zorder>widget_2</zorder>
    <zorder>headerlabel_9</zorder>
    <zorder>label_3</zorder>
    <zorder>decimalspinbox_2</zorder>
    <zorder>ui20mAPullup</zorder>
    <zorder>uiDebounce</zorder>
    <zorder>combobox_2</zorder>
    <zorder>uiTriggerIcon_2</zorder>
   </widget>
   <widget class="QWidget" name="uiTrigger3">
    <widget class="Backdrop" name="backdrop_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifiers</string>
     </property>
    </widget>
    <widget class="CheckBox" name="checkbox_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Debounce</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="CheckBox" name="checkbox_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger In</string>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::none</enum>
     </property>
    </widget>
    <widget class="CheckBox" name="checkbox_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert</string>
     </property>
     <property name="clickMarginTopSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
     <property name="clickMarginBottomSetter" stdset="0">
      <enum>CheckBox::half</enum>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>341</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255, 244, 123, 105)</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_11">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualization</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="uiTriggerIcon_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_15">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Action</string>
     </property>
    </widget>
    <widget class="ComboBox" name="combobox_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Record End Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exposure Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shutter Gating</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frame Sync Output</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
    <widget class="Label" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold: 2 volts</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="uiTriggerMotion">
    <widget class="Backdrop" name="backdrop_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="Backdrop" name="backdrop_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>21</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="Backdrop" name="backdrop_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>801</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="Backdrop" name="backdrop_8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>471</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="Button" name="uiConfigure">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
    <widget class="CheckBox" name="checkbox_14">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>320</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Debounce</string>
     </property>
    </widget>
    <widget class="CheckBox" name="checkbox_15">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>320</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert Signal</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>361</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255, 244, 123, 105)</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_12">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualization</string>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_13">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Action</string>
     </property>
    </widget>
    <widget class="ComboBox" name="combobox_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Record End Trigger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
    <widget class="QGraphicsView" name="uiTriggerIcon_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="HeaderLabel" name="headerlabel_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Area</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="ComboBox" name="uiActiveTrigger">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string comment="trig1" extracomment="trig1">Trigger 1 (TRIG1⇄)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="trig2" extracomment="trig2">Trigger 2 (TRG2⇄)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="trig3">Trigger 3 (Isolated, TRIG3±)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="motion">Motion</string>
    </property>
   </item>
  </widget>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>14</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>801</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="Label" name="uiUnsavedChangesWarning">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>397</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>* you have unsaved changes</string>
   </property>
  </widget>
  <widget class="Button" name="uiTriggerDelay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Delay</string>
   </property>
   <property name="clickMarginRightSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiRecordMode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Mode</string>
   </property>
   <property name="clickMarginLeftSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiSave">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="clickMarginRightSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="clickMarginLeftSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>uiTriggerScreens</zorder>
  <zorder>headerlabel</zorder>
  <zorder>uiActiveTrigger</zorder>
  <zorder>uiUnsavedChangesWarning</zorder>
  <zorder>uiTriggerDelay</zorder>
  <zorder>uiRecordMode</zorder>
  <zorder>uiSave</zorder>
  <zorder>uiDone</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>check_box</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combo_box</header>
  </customwidget>
  <customwidget>
   <class>DecimalSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>decimal_spin_box</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
